### ISSUE TITLE
`plot_posterior_predictive()` fails for `full_ddm` model 

### ISSUE THREAD

**Issue by jainraj** [lnccbrown/HSSM#406]
**Describe the bug** 
`plot_posterior_predictive()` for `full_ddm` model type fails with following message: 

```
The list of parameters in `list_params` ['v', 'a', 'z', 't', 'sv', 'sz', 'st'] is different from the model config in SSM Simulators (['v', 'a', 'z', 't', 'sz', 'sv', 'st']).
```

**HSSM version**
0.2.0

**To Reproduce**
```
import hssm
hssm.set_floatX('float32')
v_true, a_true, z_true, t_true = [0.5, 1.5, 0.5, 0.5]
data = hssm.simulate_data(
    model="ddm",
    theta=[v_true, a_true, z_true, t_true],
    size=1000,
)
ddm_model = hssm.HSSM(
    data=data,
    model='full_ddm',
    p_outlier=0.05,
)
ddm_model.sample(chains=1, draws=10, tune=10)
ddm_model.plot_posterior_predictive()
```

**Screenshots**
<img width="1114" alt="Screenshot 2024-05-01 at 12 29 38" src="https://github.com/lnccbrown/HSSM/assets/44366505/e6fdc374-2b8a-4f83-9411-8139841009f1">

**Additional context**
<img width="1114" alt="Screenshot 2024-05-01 at 12 31 17" src="https://github.com/lnccbrown/HSSM/assets/44366505/452ce8f7-60ba-4ac8-8e56-a94d4d270464">

In the line `params != ssms_model_config[model_name]["params"]` should we be checking sets instead of lists? 

Same issue with `plot_quantile_probability()`

**Comment by jainraj** [comment]
Also, any suggestions to get unblocked as of now?

**Comment by AlexanderFengler** [comment]
Hey @jainraj ,

thank you for using HSSM. I think, since we are dealing with lists here, 

```The list of parameters in `list_params` ['v', 'a', 'z', 't', 'sv', 'sz', 'st'] is different from the model config in SSM Simulators (['v', 'a', 'z', 't', 'sz', 'sv', 'st']).```

I can see that `sv` and `sz` are flipped. 

Will try to fix this by tonight, this should be typo level error in the code, thank you for bringing this to our attention.

**Comment by AlexanderFengler** [comment]
running into some new issue with tests have to figure that one out first, sorry or the delay.
